packages/core/src/connectivity/scp-cf/environment-accessor.ts

```diff
@@ -110,6 +110,23 @@ export function getService(service: string): Service | undefined {
   return services[0];
 }
 
+/**
+ * Get destination service if one is present.
+ *
+ * @returns Destination service
+ * @throws Error in case no destination service is found in the VCAP variables
+ */
+export function getDestinationService() {
+  const destinationService = getService('destination');
+
+  if (!destinationService) {
+    throw Error(
+      `Failed to fetch destination "${this.name}"! No binding to a destination service found.`
```

Comment:
What is `this.name` in this context? Is this something that is left from a copy-paste?

Reply:
copy paste error